### PR TITLE
Add OWNERS file

### PR DESCRIPTION
**Pull request by simonpasquier**

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+component: "Monitoring"
+
+reviewers:
+- LiliC
```
Comment (lilic) [inline OWNERS]: It's actually: lilic, it doesn't matter that much just maybe for next time :)

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- paulfantom
+- s-urbaniak
+- simonpasquier
```
Comment (paulfantom) [inline OWNERS:7]: we should add @dgrisonnet too

Comment (simonpasquier) [reply OWNERS:7]: we can't because he's not part of the OpenShift org yet

File: OWNERS (hunk 1 above)
Comment (simonpasquier) [reply OWNERS]: fixed, it matters.

**Comment by simonpasquier** [comment]
@bparees would you able to merge this PR for us?

**Pull request merged**